### PR TITLE
Ek fixes

### PR DESCRIPTION
**Pull request by webdeveric**
This fixes some horrendous HTML.

I also updated the Rizzo includes for head and body, which necessitated updating some other things.

### CODE REVIEW

File: _includes/body-footer.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <div class='wrapper--footer js-wrapper'>
-<div class='row row--sitemap' id='js-footer-nav'>
-<div class='row__inner row__inner--footer row--sitemap__inner'>
+<nav aria-label='Sitemap' class='row row--sitemap' id='js-footer-nav' itemscope='itemscope' itemtype='http://schema.org/SiteNavigationElement' role='navigation'>
```
Comment (thomasthesecond) [inline _includes/body-footer.html:2]: Should `js-footer-nav` be a class? I only ask because the `js-*` is a class on other elements.

Comment (webdeveric) [inline _includes/body-footer.html:2]: This content comes from Rizzo so you'd have to check that repo to see where this is being used.

File: _includes/head.html [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+<meta charset='utf-8'>
+<meta content='Lonely Planet' name='author'>
+<meta content='#1D508D' name='theme-color'>
+<meta content='111537044496' name='fb:app_id'>
+
+<link href='http://assets.staticlp.com' rel='dns-prefetch'>
+<link href='https://www.lonelyplanet.com/favicon.ico' rel='icon' type='image/x-icon'>
+<meta content='width=device-width, initial-scale=1' name='viewport'>
+<meta content="authenticity_token" name="csrf-param" />
+<meta content="22s8tfGrziWarwQvaWlyRU6nwIEoHRFlZy0h76drqHI=" name="csrf-token" />
```
Comment (thomasthesecond) [inline _includes/head.html:11]: Should closing slashes be removed on 10 and 11 to be consistent?

Comment (webdeveric) [inline _includes/head.html:11]: This is from Rizzo too.

Comment (thomasthesecond) [inline _includes/head.html:11]: Good to know.

**Comment by thomasthesecond** [comment]
Looks good to me.

**Pull request merged**